frontend/src/app/shared/enum/employment-state.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum EmploymentState {
+  EXMEMBER = 'EX_MEMBER',
```

Comment:
```suggestion
  EX_MEMBER = 'EX_MEMBER',
```
Either use PascalCase or SCREAMING_SNAKE_CASE.